armi/reactor/components/component.py

```diff
@@ -461,6 +470,10 @@ def clearCache(self):
         If there is a parent container and that container contains a DerivedShape, then that must be
         updated as well since its volume may be changing.
 
+        .. impl:: Clear cache after a dimenions updated
+            :id: I_ARMI_COMP_VOL1
+            :implements: R_ARMI_COMP_VOL
+
```

Comment:
Delete this one.

This requirement will need to go in a few different places in this class.